test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -196,74 +179,137 @@ describe('IexecPoco1', () => {
             expect(await iexecPoco.frozenOf(scheduler.address)).to.equal(schedulerStake);
```

Comment:
`expect(await iexecPoco.frozenOf(scheduler.address)).to.equal(schedulerFrozenBefore + schedulerStake);`
(for all frozen checks like in PR bottom)